src/main/scala/nl/knaw/dans/easy/license/app/ServiceStarter.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.license.app
+
+import org.apache.commons.daemon.{Daemon, DaemonContext}
+import org.slf4j.{Logger, LoggerFactory}
+
+class ServiceStarter extends Daemon {
+  var log: Logger = _ // Not loading logger yet, to avoid possibility of errors before init is called
```

Comment:
I don't think so. We don't want any code to execute when the `ServiceStarter` class is loading, so I guess our best option is to keep it the way it is.